test/Test/Main.purs

```diff
@@ -0,0 +1,11 @@
+module Test.Main where
+
+import Prelude
+
+import Effect (Effect)
+import Effect.Class.Console (log)
+
+main :: Effect Unit
+main = do
+  log "Test suite not yet implemented"
```

Comment:
I don't think we need an unimplmented test suite for Erumu. We can just remove it

Reply:
Removed in [`f2d9d02`](https://github.com/flipstone/purescript-erumu/pull/1/commits/f2d9d02721a381d21638012524a5baf2ffe2779f).